### ISSUE TITLE
Failing to create and API on Windows

### ISSUE THREAD

**Issue by chetanzope** [bentoml/BentoML#333]
**Describe the bug**
Failing to create an api on windows , 
`bentoml serve {saved_path}`

**To Reproduce**
1.  Failing with an error of the  fnctl  was not found 
2. fixed it by using this and creating a fcntl.py 
```
def fcntl(fd, op, arg=0):
    return 0

def ioctl(fd, op, arg=0, mutable_flag=True):
    if mutable_flag:
        return 0
    else:
        return ""

def flock(fd, op):
    return

def lockf(fd, operation, length=0, start=0, whence=0):
    return
```
3. Then the  import pwd started failing , for which I dont have any work around till now .

**Environment:**
 - OS: [windows 10 ]
 - Python/BentoML Version [Python3.6]

**Additional context**

**Comment by parano** [comment]
Thanks for reporting this issue @chetanzope ! We are working on a fix now, it was because we imported gunicorn which was only supposed to be used when running API server in linux/docker environment.

Will update with PR soon, and it should go into next release, expecting early next week.

**Comment by parano** [comment]
The fix has been merged, will update here again when the next release is out. Feel free to check out the master branch if you'd like to try out the fix. @chetanzope

**Comment by ijbo** [comment]
Appreciate !!!!

**Comment by parano** [comment]
@ijbo @chetanzope the fix is now in release 0.4.4, feel free to upgrade BentoML with `pip install -U bentoml`!

**Comment by harashm** [comment]
Hi, it seems this bug is back in latest version (0.7.6) :-(

**Comment by parano** [comment]
hi @harashm, yes we noticed that too - it was reported here: https://github.com/bentoml/BentoML/issues/663

And we have fixed the issue as well as adding CI test to prevent this from happening in the future: https://github.com/bentoml/BentoML/commit/746b16899acbb36bbdfacac3fae16609ebd015ad

The fix will be released the coming Monday in version 0.7.7

Thanks!

**Comment by parano** [comment]
hi @harashm, just want to let you know, BentoML 0.7.7 is released now, windows support have been fixed.
